test/extended/cli/set.go

```diff
@@ -0,0 +1,79 @@
+package cli
+
+import (
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	appsv1 "github.com/openshift/api/apps/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+const deploymentRunTimeout = 5 * time.Minute
+const deploymentChangeTimeout = 30 * time.Second
+
+var _ = g.Describe("[cli][Slow] oc set", func() {
+	defer g.GinkgoRecover()
+
+	var (
+		oc                      = exutil.NewCLI("oc-set", exutil.KubeConfigPath())
+		simpleDeploymentFixture = exutil.FixturePath("testdata", "deployments", "deployment-simple.yaml")
+		imageSourceLabel        = exutil.ParseLabelsOrDie("name=deployment-simple")
```

Comment:
I searched `imageSourceLabel` appears in test/extended/builds/image_source.go
For var name, please name it so that the var name is close to the test context.

Reply:
thanks, updated.